convert-hf-to-gguf.py

```diff
@@ -3128,11 +3128,11 @@ def main() -> None:
         if args.vocab_only:
             logger.info("Exporting model vocab...")
             model_instance.write_vocab()
-            logger.info("Model vocab successfully exported.")
+            logger.info(f"Model vocab successfully exported to {fname_out}.")
         else:
             logger.info("Exporting model...")
             model_instance.write()
-            logger.info("Model successfully exported.")
+            logger.info(f"Model successfully exported to {fname_out}.")
```

Comment:
This should use the name from the model, because `fname_out` otherwise could contain an unexpanded outtype template (like in its default value).

Also, the trailing dot can be confused as being part of the file name.
```suggestion
            logger.info(f"Model successfully exported to {model_instance.fname_out}")
```

The same comment also applies to the other changed line.